fathom/src/surface/elaboration.rs

```diff
@@ -926,6 +926,28 @@ impl<'interner, 'arena> Context<'interner, 'arena> {
         (name, expr)
     }
 
+    /// Elaborate a list of parameters, pushing them onto the context.
+    fn synth_and_push_params(
+        &mut self,
+        params: &[Param<ByteRange>],
+    ) -> Vec<(ByteRange, Plicity, Option<StringId>, core::Term<'arena>)> {
+        self.local_env.reserve(params.len());
+
+        Vec::from_iter(params.iter().map(|param| {
```

Comment:
Why `Vec::from_iter` over the more familiar `.collect()`?

Reply:
It formats nicer hah.

Reply:
:shrug: personal preference. I don't mind either way